Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,24 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("Type  your First name: ");
+            string firstName = Console.ReadLine();
+            Console.WriteLine("Type your Last Name");
+            string lastName = Console.ReadLine();
+            Console.WriteLine("Type your age: ");
+            int age = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Type your Weight");
+            float weight = float.Parse(Console.ReadLine()); 
+            Console.WriteLine("Type your Height");
+            double height = float.Parse(Console.ReadLine());
+
+            Console.WriteLine(firstName +" " + lastName +" is " + age +" years old, his weight is " + weight + " and his height is " + height);
+
+            double bmi = Convert.ToDouble(weight / (height * height)) * 1000;
```

Comment:
You don't need to convert this to double, float is okay. If you wanted double, you could have converted the variables above to double instaed.

Reply:
> Overall, it's a good first try :)
> 
> In the future, try to be more conscious on the requirements and don't do unnecessary things (like double conversion).
> Also, try putting a blank line between unrelated blocks of code, to indicate a separation.

A blank line helps, because code becomes more readable, less of a blob.